### DISCUSSION TITLE
How to start

### DISCUSSION THREAD

**Discussion by izark1** [open-telemetry/opentelemetry-collector-contrib#2542] [Q&A] [answered]
Hi,

I'm a Splunk technical consultant. I'm reading about open telemetry and I reached to this project. I want to collect metrics and send them via Splunk hec functionality. but how to start ?!! really I'm lost here in GitHub, what should i do to compile/run this project ? where is the beginning in these hundreds of files and scripts? I'm very new to GitHub staff.

**Comment by bogdandrutu** [comment, marked as answer]
@izark1 try to use https://github.com/signalfx/splunk-otel-collector
